target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -38,6 +38,7 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "6@eth0"
 		;;
+	wavlink,wl-wn577a2|\
 	buffalo,wcr-1166ds)
```

Comment:
Please change the order here, so alphabetic sorting is maintained:
	buffalo,wcr-1166ds|\
	wavlink,wl-wn577a2)

Reply:
ACK